src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -3559,14 +3559,14 @@ void ObjectMgr::LoadPlayerInfo()
     {
         uint32 oldMSTime = getMSTime();
 
-        _playerXPperLevel.resize(sXpGameTable.GetTableRowCount() + 1, 0);
+        _playerXPperLevel.resize(sXpGameTable.GetTableRowCount() - 1, 0);
```

Comment:
this change is wrong, you will access `_playerXPperLevel[sXpGameTable.GetTableRowCount()]`